inductiva/resources/machines_base.py

```diff
@@ -9,12 +10,18 @@
 from inductiva.client.apis.tags import compute_api
 
 
+class ResourceType(enum.Enum):
+    """Enum to represent the type of machine to be launched."""
+
+    STANDARD = "standard"
+    MPI = "mpi"
+
+
```

Comment:
Use a `StrEnum`; its members can be used as string wherever strings are needed:

```suggestion
class ResourceType(enum.StrEnum):
    """Enum to represent the type of machine to be launched."""

    STANDARD = "standard"
    MPI = "mpi"


```

Reply:
The reason I didn't follow this suggestion is because the StrEnum is only available from Python 3.11.